Source/Core/SwipeActions.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  Swipe.swift
+//  Eureka
+//
+//  Created by Marco Betschart on 14.06.17.
+//  Copyright © 2017 Xmartlabs. All rights reserved.
+//
+
+import Foundation
+
+public typealias SwipeActionHandler = (ContextualAction, ContextualActionSource, ((Bool) -> Void)?) -> Void
```

Comment:
I think we should pass the row as closure argument.

Reply:
`ContextualActionSource` can be removed since the row is passed as argument now.

Reply:
That's a brilliant idea! We should ideally use the specific Row type right? Not just `BaseRow`.

Reply:
Implemented as `public typealias SwipeActionHandler = (ContextualAction, BaseRow, ((Bool) -> Void)?) -> Void`